### PR TITLE
tkt-82308: Implement interface choices methods

### PR DESCRIPTION
**Pull request by william-gr**

### CODE REVIEW

File: src/middlewared/middlewared/plugins/network.py [hunk 1]
```diff
@@ -1268,38 +1268,105 @@ def __convert_aliases_to_datastore(self, data):
...
                     choices.pop(interface['vlan_parent_interface'], None)
                     continue
         return choices
 
+    @accepts(Str('id', null=True, default=None))
+    async def bridge_members_choices(self, id):
+        """
+        Return available interface choices for `bridge_members` attribute.
+
+        `id` is the name of the bridge interface we want to update or null in case its a new
```
Comment (Mrt134) [inline src/middlewared/middlewared/plugins/network.py]:
```suggestion
        `id` is the name of the bridge interface to update or null for a new
```

File: src/middlewared/middlewared/plugins/network.py [hunk 2]
```diff
@@ -1268,38 +1268,105 @@ def __convert_aliases_to_datastore(self, data):
...
+            'include': include,
+        })
+        return choices
+
+    @accepts(Str('id', null=True, default=None))
+    async def lag_ports_choices(self, id):
+        """
+        Return available interface choices for `lag_ports` attribute.
+
+        `id` is the name of the LAG interface we want to update or null in case its a new
```
Comment (Mrt134) [inline src/middlewared/middlewared/plugins/network.py]:
```suggestion
        `id` is the name of the LAG interface to update or null for a new
```

**Comment by william-gr** [comment]
@Mrt134 committed, thanks

**Comment by william-gr** [comment]
@freenas/middleware please someone get to this ASAP, its blocking UI work.

**Pull request merged**